ini/stm32f1.ini

```diff
@@ -332,17 +332,15 @@ upload_protocol = serial
 platform                    = ${common_stm32.platform}
 extends                     = stm32_variant
 board                       = genericSTM32F103VE
-board_build.variant         = MARLIN_F103Vx
+board_build.variant         = MARLIN_F103VE_LONGER
 board_build.rename          = project.bin
 board_build.offset          = 0x10000
 board_upload.offset_address = 0x08010000
-build_flags                 = ${stm32_variant.build_flags}
-                              -DMCU_STM32F103VE -DU20 -DTS_V12 -DLED_BUILTIN=PC2 -UPIN_WIRE_SDA
-                              -UPIN_WIRE_SCL -DPIN_WIRE_SDA=PB11 -DPIN_WIRE_SCL=PB10
-                              -DHAL_DAC_MODULE_DISABLED -DHAL_I2S_MODULE_DISABLED
-build_unflags               = ${stm32_variant.build_unflags}
-                              -DUSBCON -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
+build_flags                 = ${stm32_variant.build_flags} -DMCU_STM32F103VE -DU20 -DTS_V12
+build_unflags               = ${stm32_variant.build_unflags} -DUSBCON -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
 extra_scripts               = ${stm32_variant.extra_scripts}
+monitor_speed               = 250000
```

Comment:
CH340 automatic baudrate

Reply:
The PlatformIO Serial Monitor is invoked within VSCode with the command `platformio-ide.serialMonitor` and it gets the `monitor_speed` from the global environment, or the `default_envs`. Currently the Auto Build Marlin extension just sends the `platformio-ide.serialMonitor` command to VSCode, but ABM could be modified to instead use `pio device monitor --baud ...` and get the `monitor_speed` from the pertinent environment.

Reply:
pio device list
/dev/ttyUSB0
------------
Hardware ID: USB VID:PID=1A86:7523 LOCATION=3-3
Description: USB2.0-Serial

which is the onboard CH340G chip.. is there some way to set/filter vid/pid ?

Reply:
I expect you'd have to embed `pio` commands into a script to more smartly list connections. AFAIK the monitor will just connect to the first thing that looks like an embedded device unless you are more specific.

Reply:
yea it works here... "pio device monitor" its just in case you have 2 serial usb devices ;)

Reply:
`pio device monitor --baud 250000`?

This command might not get the baud rate from the `platformio.ini` => `default_envs`.